src/test/java/org/jenkinsci/plugins/workflow/multibranch/ReadTrustedStepTest.java

```diff
@@ -259,6 +260,7 @@ public void symlinksInReadTrustedCannotEscapeWorkspaceContext() throws Exception
     @Issue("SECURITY-2491")
     @Test
     public void symlinksInUntrustedRevisionCannotEscapeWorkspace() throws Exception {
+        assumeFalse(Functions.isWindows()); // On Windows, the symlink is treated as a regular file, so there is no vulnerability, but the behavior is different.
```

Comment:
In other words, the contents of the symlink itself get read, not the contents of the file that it links to.